demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -897,7 +900,7 @@ def validate_modeling_rule(
                         err, schema_test_case, modeling_rule_test_suite
                     )
         else:
-            err = f"Schema file does not exist in path {modeling_rule.schema_path}"
+            err = f"Schema file does not exist in path {get_relative_path_to_content(modeling_rule.schema_path)}"
```

Comment:
If it's not there, will `get_relative_path_to_content` work?

Reply:
yes, tested the function now. good thought!